Misc/NEWS.d/next/Core and Builtins/2021-05-26-19-10-47.bpo-43693.1KSG9u.rst

```diff
@@ -0,0 +1,6 @@
+The opargs for cell-related opcodes were out of sync with the fast locals,
```

Comment:
I don't follow this. What do you mean by "out of sync".

Maybe replace this with something like:
"Computation of the offsets of cell variables is done in the compiler instead of at runtime. This reduces the overhead of handling cell and free variables, especially in the case where a variable is both an argument and cell variable.
A new opcode ``MAKE_CELL`` has been added to convert arguments, and other locals, to cell variables on function entry."